src/test/java/com/microsoft/sqlserver/jdbc/TestResource.java

```diff
@@ -200,5 +200,6 @@ protected Object[][] getContents() {
             {"R_AKVProviderNull", "The Azure key store provider is null."},
             {"R_objectNullOrEmpty", "The {0} is null or empty."},
             {"R_cekDecryptionFailed", "Failed to decrypt a column encryption key using key store provider: {0}."},
-            {"R_connectTimedOut", "connect timed out"}};
+            {"R_connectTimedOut", "connect timed out"},
+            {"R_sessionKilled", "Cannot continue the execution because the session is in the kill state"}};
```

Comment:
Reminder to copy over localizations from main to the servicing branch in the VSO repo.